central/declarativeconfig/watch_handler.go

```diff
@@ -93,7 +104,9 @@ func (w *watchHandler) OnStableUpdate(val interface{}, err error) {
 }
 
 func (w *watchHandler) OnWatchError(err error) {
-	log.Errorf("Error watching declarative configuration directory: %v", err)
+	if !errors.Is(err, os.ErrNotExist) {
```

Comment:
Hmm, did we see `os.ErrNotExist` before here?

Reply:
Yesh I noticed it came up when running the test so I fixed it. Happy to revert in case you think its not necessary.